draft-kazuho-httpbis-priority.md

```diff
@@ -210,20 +204,23 @@ there is not a clean mapping to HTTP/3.
 The extensible priority scheme is negotiated using the described mechanism. It is
 identified by the 8-bit value of 2.
 
+Endpoints using this scheme SHOULD emit the Priority header field regardless of
```

Comment:
I think this PR accidentally reverted this change?  I can fix that....

Reply:
I think that is the case.

Reply:
Reverted.